src/Inventory.zig

```diff
@@ -339,12 +342,9 @@ pub const Sync = struct { // MARK: Sync
 					inventory.inv._items[inventory.inv._items.len - 1].amount = recipe.resultAmount;
 					inventory.inv._items[inventory.inv._items.len - 1].item = .{.baseItem = recipe.resultItem};
 				},
-				.other => {},
+				.blockInventory, .other => {},
```

Comment:
Please make this a separate branch with a todo, since this is where you'd need to load it.